tensorboard/default.py

```diff
@@ -86,4 +87,30 @@ def get_plugins():
 
   :rtype: list[Union[base_plugin.TBLoader, Type[base_plugin.TBPlugin]]]
   """
+
   return _PLUGINS[:]
+
+# TODO(stephanwlee): Combine with get_plugins when concept of first-party
+# plugins is undone.
+def get_dynamic_plugins():
+  """Returns a list specifying TensorBoard's dynamically loaded plugins.
+
+	A dynamic TensorBoard plugin is specified using entry_points [1]. This will be
+  the de facto mechanism to load plugins and there no longer will be a concept
+  of the first-party plugins as depicted above in `get_plugins`.
```

Comment:
Hmm, do we really want to ~deprecate `get_plugins`? Users who launch
TensorBoard via a custom `tensorboard.program.TensorBoard` instance may
continue to want a fixed list of plugins rather than using the entry
points mechanism—perhaps because they build/package Python programs in
such a way that entry points are not used, or perhaps because they
simply want to have clear build-time dependencies for a production
system.